.github/workflows/release.yml

```diff
@@ -118,38 +117,12 @@ jobs:
           echo "Publish docs to Confluence: $PUBLISH_DOCS"
```

Comment:
- The `Append Fix Notes to Confluence` step includes formatting the `new_fix_notes` but ensure it does not introduce unintended issues or break the existing Confluence content format.